projects/models.py

```diff
@@ -101,14 +105,30 @@ def __str__(self):
         return self.title
 
 
+def add_project_controls(instance: Project):
+    try:
+        controls = apps.get_model("catalogs", "Controls")
+        catalog_controls = controls.objects.filter(catalog_id=instance.catalog)
+        print(catalog_controls)
```

Comment:
I assume this was for debugging, but we should take it out before merging. Or switch it to a log

Reply:
Yep